### ISSUE TITLE
Ball trail visible when ball under playfield

### ISSUE THREAD

**Issue by francisdb** [vpinball/vpinball#1175]
**Describe the bug**

![image](https://github.com/vpinball/vpinball/assets/161305/79b59a74-4c9c-4808-8e15-c1a5736dd37a)

**To Reproduce**
1. Launch the ball and when it bounces under top playfield watch the trails

If the bug appears when playing a specific table, give a link to a place where the exact version of the table file can be downloaded.

Alien Nostromo Ultimate Edition 4.0 
https://www.vpforums.org/index.php?app=downloads&showfile=16529

**Expected behavior**

Ball not visible, ball trail not visible

**Versions & Settings**
 - VPX: master / GL
 - default config
 - AMD Radeon
 - Linux

**Comment by francisdb** [comment]
Different table, same issue

![image](https://github.com/vpinball/vpinball/assets/161305/563df260-8bb5-4049-9b03-0662fe688e92)

**Comment by passionforpins** [comment]
It's a depth bias issue on some tables, according to Apophis. I noticed if ball trails are ON this happens on many tables I tested. I made a short video on Bad Cats. I back tracked VPX versions and it started happening somewhere between 1732 and 1746. 
https://drive.google.com/file/d/1qLBHqgiDJjW6UGNI0kaH5lqQBbQ7kXIK/view?usp=sharing

I noticed the same thing on Cyclone except it flys visibly under the apron and same on Defender.
